_LowLevelCode/CPP/cpp_communicator/cpp_communicator.cpp

```diff
@@ -100,23 +187,25 @@ void mexFunction(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[])
     } // end case
     } // end switch
 
-     if (nlhs > 0)
-         plhs[(int)labIndex_Out::comm_ptr] = pCommunicatorHolder->export_hanlder_toMatlab();
-
+    if (nlhs > 0)
+        plhs[(int)labIndex_Out::comm_ptr] = pCommunicatorHolder->export_hanlder_toMatlab();
```

Comment:
missing `{ }`

Reply:
Its the school in Mantid not using brackets in loops and ifs for single line expression
I was critical to it initially, but get used to it. 
not sure, should we request the brackets or not -- would agree to common desision

Reply:
I'll +1 for always using brackets. It leads to fewer mistakes if another developer should add any lines to the loop further down the line.

Reply:
done